packages/rs-drive-abci/src/execution/engine/initialization/init_chain/v0/mod.rs

```diff
@@ -79,6 +79,13 @@ where
 
         state_guard.set_genesis_block_info(Some(genesis_block_info));
 
+        // Store initial protocol version
+        self.drive.store_current_protocol_version(
+            state_guard.current_protocol_version_in_consensus(),
```

Comment:
won't this just be the default?

Reply:
Yeah, we should use genesis param instead of default.